pkg/role/role.go

```diff
@@ -17,6 +17,9 @@ const (
 	// Observer indicates that this node observes batch writes to the ledger and persists document operations to the DB
 	Observer = "sidetree-observer"
 
+	// Monitor indicates that this node periodically monitors the blocks and checks that all document operations have been persisted to the DB
```

Comment:
what happens if some operation wasn't persisted?

Reply:
The Monitor periodically checks the blocks and if it finds that an operation that was supposed to be persisted wasn't persisted, it will persist the operation.